docs/apache-airflow/start.rst

```diff
@@ -110,7 +110,7 @@ run the commands below.
     # run your first task instance
     airflow tasks test example_bash_operator runme_0 2015-01-01
     # run a backfill over 2 days
-    airflow dags backfill example_bash_operator \
+    airflow backfill create --dag example_bash_operator \
```

Comment:
```suggestion
    airflow backfill create --dag-id example_bash_operator \
```

Reply:
done in the followup pr https://github.com/apache/airflow/pull/43478